coco/check.py

```diff
@@ -71,7 +71,7 @@ async def on_failure(self, hosts=None):
             logger.debug(f"Calling {self.on_failure_call} because {self._name} failed.")
             result.embed(
                 self.on_failure_call,
-                await self.forwarder.internal(self.on_failure_call, "", {}),
```

Comment:
Method got switched from `""` to `None`, do you need to do `if method is None: method = ""` anywhere? Or do the methods all know how to deal with `method` having a value of `None`?

Reply:
I don't think the `""` was used at all. `internal` ignores `method`